### ISSUE TITLE
fluentbit 1.3.2 not working on windows

### ISSUE THREAD

**Issue by lkjangir** [fluent/fluent-bit#1728]
## Bug Report

**Describe the bug**
The latest release 1.3.2 is not working on Windows. It is not starting up and failing up with some _DLL missing error_

**To Reproduce**
-  Require Windows server (Any flavour )
- Fluent-bit installer 

```
Error messages - 
The program can't start because SSLEAY32.dll is missing from your computer. 
same error for another DLL -            LIBEAY32.dll
again for one more DLL -                  VCRUNTIME140.dll

```
- Steps to reproduce the problem:
- Install fluent-bit installer http://fluentbit.io/releases/1.3/td-agent-bit-1.3.2-1.AMD64.exe on Windows 2016  server
- run command `/path/fluent-bit.exe -i winlog -p 'Channels=Security' -o stdout`

**Screenshots**
![image](https://user-images.githubusercontent.com/4569136/68671568-96f25d80-0575-11ea-8c8b-ec8617fc56e3.png)

![image (1)](https://user-images.githubusercontent.com/4569136/68671627-ba1d0d00-0575-11ea-92f7-64605ce9a1b8.png)

**Your Environment**
* Version used: 1.3.2
* Configuration: used CLI and configuration file as well.
* Environment name and version: Windows 
* Server type and version: Windows Server 2016

**Comment by fujimotos** [comment]
> The latest release 1.3.2 is not working on Windows. It is not starting up and failing up with some DLL missing error

This is due to Fluent Bit being dependent on openssl. You need to install `ssleay32.dll`
and `libeay32.dll` manually on your system to run Fluent Bit (at least as of v1.3.2)

I've posted a fix for the issue at #1656, but it's not merged yet.

**Comment by fujimotos** [comment]
This is already fixed as of v1.4.6. Close this issue for now.
